Jellyfin/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "alexbcberio",
+    "id": "202915432175239169"
+  },
+  "url": "jellyfin.org",
+  "description": {
+    "en": "The Free Software Media System"
+  },
+  "service": "Jellyfin",
+  "version": "1.0",
+  "logo": "https://raw.githubusercontent.com/jellyfin/jellyfin/master/Emby.Dlna/Images/logo240.png",
+  "thumbnail": "https://jellyfin.org/favicon.ico",
+  "color": "#00698c",
+  "category": "videos",
+  "tags": [
+      "video",
+      "movie",
+      "tv",
+      "media"
+  ],
+  "regExp": "([a-z0-9]+[.])*([a-z0-9]+)"
```

Comment:
Please make sure other presences still work, because you put *

Reply:
Other presences worked previously Ok, I've changed the way the presence is initialized in [845dbfeb8bfd17c1cc4d9b7543e16ddcd99ec05e](https://github.com/alexbcberio/Presences/commit/845dbfeb8bfd17c1cc4d9b7543e16ddcd99ec05e).

In the new commit the presence is only initialized if we are in the official page or we are in the web client, by accessing a variable on the localstorage that keeps the last log in timestamp and verifying that it has been in the previous 30 seconds.